README.md

```diff
@@ -10,14 +10,20 @@ Both are created, run and managed by ReDI School of Digital Integration. We're a
 ## Getting started for developers
 
 1. Make sure [MongoDB](https://docs.mongodb.com/manual/installation/) is installed on your computer.
-2. If you're re-seeding, drop the database files: `rm -rf mongodb-data/*` and run `yarn seed` to seed the database.
-3. Make sure you're running the _v14.17.6_ version of Node locally when you're running the backend server. To do this, you can use [nvm](https://github.com/nvm-sh/nvm#node-version-manager), which allows you to select different versions of Node via the command line. Alternatively, we have added support for [Volta](https://docs.volta.sh/guide/understanding). So if you choose, you can use Volta, which sets project-defined tools automatically.
-4. Run `yarn` in project root to install dependencies.
-5. Run `yarn start:all` to boot all apps, _or_ a subset of apps using the `start:x` commands in package.json.
+2. Make sure you're running the _v14.17.6_ version of Node locally when you're running the backend server and checkout to a new branch. To do this, you can use [nvm](https://github.com/nvm-sh/nvm#node-version-manager), which allows you to select different versions of Node via the command line. Alternatively, we have added support for [Volta](https://docs.volta.sh/guide/understanding). So if you choose, you can use Volta, which sets project-defined tools automatically.
+3. Run `yarn` in project root to install dependencies.
+4. Run `yarn start:all` to boot all apps, _or_ a subset of apps using the `start:x` commands in package.json.
 
    It'll take a while and lots of warnings will show until everything's booted.
 
-6. See the [Onboarding Checklist](https://github.com/talent-connect/connect/wiki#onboarding-checklist) in our Wiki. 
+5. See the [Onboarding Checklist](https://github.com/talent-connect/connect/wiki#onboarding-checklist) in our Wiki.
+6. If you're re-seeding, drop the database files: `rm -rf mongodb-data/*` and run `yarn seed` to seed the database.
+   Alternatively, you can drop the database in the MongoDB Shell:
+   in a new terminal window run the following
+   `mongo`
+   `use mongodb`
+   `db.dropDatabase()`
+   Then run `yarn seed` in project root to seed the database.
```

Comment:
Can we move this bullet in-between the current items #3 and #4? It's logical to (re)seed after step #3.

Reply:
Okay, I thought to list it as the last point because we don't do re-seeding often. 
I also inserted the correct link for 'Workflow for design tasks' in 'Getting started for designers'.